tests/unit/test_rat_board.py

```diff
@@ -132,3 +135,12 @@ async def test_modify_rescue_explosion(rat_board_fx, random_string_fx):
             raise RuntimeError  # intentionally raise an exception
 
     assert random_string_fx in rat_board_fx, "the board dropped the rescue!"
+
+@pytest.mark.asyncio
+async def test_modify_rescue_datetimelastcase(rat_board_fx, monkeypatch):
+    """
+    Verifies that _datetime_last_case is being modified when creating a new case
+    """
+    pre_datetime = datetime.datetime.now(tz=timezone.utc)
```

Comment:
[error] This shouldn't pull the current datetime, but rather the current value of `rat_board_fx._datetime_last_case`

To ensure predictability, one might have to insert a small arbitrary delay to make the following assertion work.

Reply:
Added the sleep, but it shouldn't (only) pull the current version of rat_board_fx._datetime_last_case, because that value will be 1900-01-01 by default, and thus downgrade the test. I've added an additional step to the test to also check the initial value.